lib/core/security/profileRepository.ts

```diff
@@ -44,9 +71,10 @@ class ProfileRepository extends Repository {
     });
 
     this.module = securityModule;
-    this.collection = 'profiles';
-    this.ObjectConstructor = Profile;
     this.profiles = new Map();
+
+    super.collection = 'profiles';
```

Comment:
Because those variable are defined in the superclass, so I set them using super instead of creating a variable that would shadow the variable from the superclass